hedged.go

```diff
@@ -140,6 +140,11 @@ func (ht *hedgedTransport) RoundTrip(req *http.Request) (*http.Response, error)
 		switch {
 		case resp.Resp != nil:
 			resultIdx = resp.Index
+			if resultIdx == 0 {
+				ht.metrics.originalRequestWinsInc()
```

Comment:
is there a reason to record both of these? isn't one deriveable from the other?

total requests - original requests wins = hedged requests wins

Reply:
Well, yeah, just with the extra steps.

I don't have a strong opinion on that, the internal `ht.metrics` can have only 1 atomic counter but there can be 2 exposed metrics.

@dannykopping @storozhukBM wdyt?

Reply:
I'm not sure.
For example:

=> request
  => original... slow
  => hedge 1... slow
  => hedge 2.. fast => wins!

That would be 3 total requests - 0 original request wins = 3 hedged request wins...?

Reply:
WOW, good catch.